beacon-chain/blockchain/service_test.go

```diff
@@ -452,3 +515,119 @@ func TestProcessingBlockWithAttestations(t *testing.T) {
 	chainService.cancel()
 	exitRoutine <- true
 }
+
+func TestProcessingBlockWithChainSwitch(t *testing.T) {
```

Comment:
Let's just rename this to `TestProcessingBlocks`

Reply:
Done